compiler/sem/semtypes.nim

```diff
@@ -688,7 +688,10 @@ proc semIdentVis(c: PContext, kind: TSymKind, n: PNode,
       c.config.semReportIllformedAst(
         n, "Expected two nodes for postfix expression, but found " & $n.len)
   else:
-    result = getDefNameSymOrRecover(newSymGNode(kind, n, c))
+    let sym = newSymGNode(kind, n, c)
+    if sym.kind == nkError:
+      localReport(c.config, sym)
```

Comment:
```suggestion
      # XXX: replace with propagating the error
      localReport(c.config, sym)
```
The general direction is to remove `localReport`, meaning that when it is used, a comment should make it clear that this isn't "proper".